src/widgets/category/category-widget-model.js

```diff
@@ -136,10 +136,45 @@ module.exports = WidgetModel.extend({
     }
   },
 
+  sizeAcceptedCatsInData: function () {
```

Comment:
:+1: https://github.com/CartoDB/cartodb.js/compare/v4...dataview-filter-methods to make it easier to see the changes :kissing_heart: